components/application/logout.jsx

```diff
@@ -1,28 +1,28 @@
 import React from 'react'
 import { AppBar } from '@oacore/design'
 
-import { API_URL } from '../../config'
 import styles from './logout.css'
 
 import { withGlobalStore } from 'store'
 
-const LOGOUT_URL = `${API_URL.replace(
-  '/internal',
-  ''
-)}/logout?continue=${encodeURIComponent(
-  typeof window !== 'undefined'
-    ? `${window.location.origin}/?reason=logout`
-    : ''
-)}`
+const { IDP_URL } = process.env
+
+const getLogoutUrl = pathname => {
+  const url = new URL('./logout', IDP_URL)
```

Comment:
Out of curiosity. What's the point of having here dot? It looks like it has no effect at all.


```jsx
new URL('./logout', 'https://localhost:8000/test').toString() // "https://localhost:8000/logout"
```

Reply:
It would have effect with

```js
new URL('./logout', 'https://localhost:8000/test/').toString()
```